src/main/java/github/scarsz/discordsrv/objects/managers/AccountLinkManager.java

```diff
@@ -1,157 +0,0 @@
-/*-
- * LICENSE
- * DiscordSRV
- * -------------
- * Copyright (C) 2016 - 2021 Austin "Scarsz" Shapiro
- * -------------
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- * 
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public
- * License along with this program.  If not, see
- * <http://www.gnu.org/licenses/gpl-3.0.html>.
- * END
- */
-
-package github.scarsz.discordsrv.objects.managers;
-
-import org.bukkit.event.Listener;
-
-import java.util.Map;
-import java.util.Set;
-import java.util.UUID;
-
-/**
- * Class for accessing and managing linked accounts.
- */
-public abstract class AccountLinkManager implements Listener {
```

Comment:
Deprecating this is probably worth it (can even be converted back to regular class) - a lot of people use the linking system of DiscordSRV via the API